lib/Settings.tsx

```diff
@@ -77,33 +80,51 @@ export const Settings: React.FC<SettingsProps> = ({
     [toggleStats],
   )
 
+  const changeFormat = useCallback(
+    (e: ChangeEvent<HTMLSelectElement>) => {
+      onFormat(e.target.value as Format)
+    },
+    [onFormat],
+  )
+
+  const changeResolution = useCallback(
+    (e: ChangeEvent<HTMLSelectElement>) => {
+      if (e.target.value === '') {
+        onVapix('resolution', '')
+        return
+      }
+      onVapix('resolution', e.target.value)
```

Comment:
The if statement here seems a bit unnecessary?

Reply:
Indeed, would be interesting to see the history of this.

Reply:
`onVapix('resolution', '')`

Results in...

```
if (value === '') {
  delete p[key]
  return { ...p }
}
```

Reply:
I may or may not be the culprit of this less than ideal solution. :smile:

Reply:
Fixed now.